lib/msal-angular/src/msal.service.ts

```diff
@@ -124,31 +124,31 @@ export class MsalService extends UserAgentApplication {
     handleRedirectCallback(authCallback: authResponseCallback): void;
     handleRedirectCallback(authOrTokenCallback: authResponseCallback | tokenReceivedCallback, errorReceivedCallback?: errorReceivedCallback): void {
         super.handleRedirectCallback((authError: AuthError, authResponse: AuthResponse) => {
-            if (authResponse) {
+            if (authError) {
                 if (authResponse.tokenType === "id_token") {
```

Comment:
@jasonnutter The `authResponse` will not populate the `tokenType` in the case of an error. Should we collapse these 2 broadcasts into a single generic `msal:Error` or find a way to expose the type in core?

Reply:
AH, good. I think for now, we can infer it via `getAccount` (if it returns null, assume we were making a login call).